react/src/components/utils/EmptyTablePlaceholder.tsx

```diff
@@ -0,0 +1,25 @@
+import { SectionMessage } from '../../core-components';
```

Comment:
here are some comments for this component, but depending on https://github.com/TACC-Cloud/hazmapper/pull/268#discussion_r1805297011, they might not be needed.  so you might ignore this if EmptyPLaceholder is replaced by something else:

* do we want put this in `src/component/utils`?  is that spot for smaller components that will be reused by things in `src/components`?  if it is very Projects or ProjectListing and only used there, we could stick under src/components/Projects/
* the no children handling of "No data available" is never used. drop?